lang/de.xml

```diff
@@ -50,9 +50,10 @@
 		<delete>Löschen</delete>
 		<file_corrupted>Die Datei ist beschädigt.</file_corrupted>
 		<no>Nein</no>
-		<select_all>Wähle alle aus</select_all>
-		<select>Wählen</select>
-		<please_wait>Bitte warten...</please_wait>
+		<select_all>Alle auswählen</select_all>
+		<clear_all>Alle löschen</clear_all>
```

Comment:
sorry for delay, i current checking, take time, move clear_all inside content manager, is no inside common string, is inside content manager app